be/src/storage/lake/replication_txn_manager.cpp

```diff
@@ -248,7 +248,7 @@ Status ReplicationTxnManager::replicate_remote_snapshot(const TReplicateSnapshot
                                  src_snapshot_info.snapshot_path, request.src_tablet_id, request.src_schema_hash,
                                  remote_header_file_name, config::download_low_speed_time));
         TabletMeta tablet_meta;
-        auto status = tablet_meta.create_from_memory(header_file_content);
+        auto status = tablet_meta.create_from_memory(header_file_content, false);
```

Comment:
how about add a new interface to TabletSchema, it creates a schema but won't touch the TabletSchemaMap.

It will be easier to predict the behavior from interface name other than these true/false switch to tune the internal implementation behavior.

Reply:
what the new interface name should be ? I fact, the interface `create_from_memory` is newly added.

Reply:
Then just bypass adding to the tabletMetaMap in create_from_memory, and Update the comment for the interface, highlight the difference between create_from_memory and create_from_file, maybe name to create_from_memory_non_shared?

Reply:
`create_from_memory_non_shared` is unclear, its the interface of `TabletMeta`, but what is shared is the tablet schema inside it. just keep `create_from_memory` and highlight the difference using comment

Reply:
good to me.